urllib3/connectionpool.py

```diff
@@ -167,7 +172,7 @@ class HTTPConnectionPool(ConnectionPool, RequestMethods):
     def __init__(self, host, port=None, strict=False,
                  timeout=Timeout.DEFAULT_TIMEOUT, maxsize=1, block=False,
                  headers=None, retries=None,
-                 _proxy=None, _proxy_headers=None,
+                 _proxy=None, _proxy_headers=None, cj=None,
```

Comment:
I don't know if I like calling the cookie jar argument `cj`: seems like needless brevity.